src/www/ui/api/Models/ApiVersion.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/*
+  SPDX-FileCopyrightText: © 2023 Samuel Dushimimana <dushsam100@gmail.com>
+
+ SPDX-License-Identifier: GPL-2.0-only
+*/
+/**
+ * @file
+ * @brief ApiVersion enum
+ */
+namespace Fossology\UI\Api\Models;
+
+/**
+ * @class ApiVersion
+ * @brief ApiVersion enum
+ */
+class ApiVersion
+{
+  const V1 = 'v1';
+  const V2 = 'v2';
```

Comment:
Would request to have the versions as integers. It will make the system future proof and allows checking versions like bellow
```php
if ($request->getAttribute('apiVersion') > ApiVersion::V1) {
  // do something new
}
```

Reply:
Noted